### PR TITLE
Additional Factories 1

### PR DESCRIPTION
**Pull request by newsiberian**
Hello, requested by reactioncommerce/reaction#636

### CODE REVIEW

File: common/factories/cart.js [hunk 1]
```diff
@@ -19,4 +20,94 @@ Factory.define("cart", Cart, {
...
+  userId: Factory.get("anonymous")
+};
+
+Factory.define("cart", Cart, Object.assign({}, cart));
+
+Factory.define("cartToOrder", Cart, Object.assign({}, cart, cartToOrder));
+
+Factory.define("anonymousCart", Cart, Object.assign({}, cart, anonymousCart));
+
+//Factory.define("cart", Cart, {
```
Comment (aaronjudd) [inline common/factories/cart.js]: Delete please, instead of commenting.. (that's why we have source control).

**Comment by aaronjudd** [comment]
Approved!

**Comment by aaronjudd** [comment]
Merged, but not published yet. we need to increment package version, remove commented code, then publish.

**Comment by newsiberian** [comment]
Hello, oops, I'm sorry

**Pull request merged**